README.md

```diff
@@ -1,10 +1,10 @@
 # BibTags
 
-This is BibTags. A bibtex library containing many literature references to papers in computer science with a focus on variability and software product lines.
+This is BibTags, a BibTeX library that contains many curated literature references to papers in computer science, with a focus on variability and software product lines.
 
 ## How To Use
-1. Clone this repo or add as a submodule
-2. Run `./run clean`.
+1. Clone this repo or add as a submodule (e.g., `git clone https://github.com/TUBS-ISF/BibTags.git`)
```

Comment:
This suggests the command creates a submodule which it does not. Maybe add an `e.g.` for both ways to clone?